index.js

```diff
@@ -267,4 +263,13 @@ Emailer.admin = {
     }
 };
 
+Emailer.addMiddleware = function(data, callback) {
+    data.app.head('/emailer-mandrill/reply', function(req, res) {
+        res.sendStatus(200);
+    });
+    data.app.post('/emailer-mandrill/reply', Emailer.receive);
+
+    callback();
+};
+
```

Comment:
Hmmz, GitHub's reply by email doesn't seem to be the best. I'm sure we can do better than that :)

Thing is, setting the routes with the previous hook didn't seem to work. Before this change the plugin would always recommend the route `/admin/plugins/emailer-mandrill/emailer-mandrill/reply`, but this is not accessible if you're not an admin. Is there a way to expose an endpoint that can be reached by Mandrill? 
I suppose just `/emailer-mandrill/reply` worked in previous versions (pre-1.0.0)? I'm not familiar enough with NodeBB, but it still replies with either `401` or `403` if I test that route as well. Only after this PRs I can make the webhook work on `/emailer-mandrill/reply`.